pkg/runtime/runtime.go

```diff
@@ -976,7 +976,12 @@ func (a *DaprRuntime) isAppSubscribedToBinding(binding string) bool {
 		// TODO: Propagate Context
 		ctx := context.Background()
 		resp, err := a.appChannel.InvokeMethod(ctx, req)
-		return err == nil && resp.Status().Code != nethttp.StatusNotFound
+		if err != nil {
+			return false
```

Comment:
Where is the fatal log?

Reply:
I'll add it here, though I'm afraid this will bite us somehow.

Reply:
It's also worth noting that this unlikely return `err != nil` because `parseChannelResponse` in `http_channel.go` returns them as `500` instead. Would you also like it for `500` to `log.Fatal`?

FYI: I created issue #3660 to capture what users on Discord have reported.